flint.ui/src/components/Stepper/StepperGCBM.vue

```diff
@@ -22,36 +22,36 @@ export default {
 
   methods: {
     onChange(current) {
-      this.current = current;
+      this.current = current
 
-     if (current == 0) {
-       this.$router.push({
-           name: "gcbmdashboard",
-           params: { current },
-       })
-     } else if (current == 1) {
-       this.$router.push({
-           name: "gcbmupload",
-           params: { current },
-       })
-     } else if (current == 2) {
-       this.$router.push({
-           name: "gcbmrun",
-           params: { current },
-       })
-     }
-    },
-  },
-};
+      if (current == 0) {
```

Comment:
Small nitpick:
-  We can use switch-case here
- Add comments for explaining what does `0`, `1` etc. numbers signify